doc/usage.rst

```diff
@@ -604,12 +606,29 @@ are comma-separated lists.
     ID, the second label is used. The third label (optional) is used for items with both relationships covered.
     The labels in the example are the default values.
 
+:sourcetype: *optional*, *multiple arguments (space-separated)*
+
+    The list of relationships that should be used to filter the target. The relationships considered for
+    filtering are from the "Source" items to the "Target" items. In this example, if an RQT-item
+    is not linked to a TEST-item with *validated_by* and/or *covered_by*, this source item will be labeled as
+    *uncovered*.
+
+:targettype: *optional*, *multiple arguments (space-separated)*
+
+    The list of relationships that should be used to filter the nested target. The relationships considered for
+    filtering are from the "Target" items to the "Target-of-target" items. These relationhips will also be used to label
+    additional slices if the *<<attribute>>* option that accepts multiple arguments is unused.
+    In this example, if a TEST-item is not linked to a TEST_REP-item with one or more of
+    *passed_by/skipped_by/failed_by*, the source item will be labeled as *covered* instead of
+    *passes*, *skipped* or *fails*, which are the human readable and reversed
+    forms of the arguments for this option. If an RQT-item with multiple TEST-items, one linked to
```

Comment:
This sentence is incomplete

Reply:
b58d73fcd56277b33de064033fef52ed4c799edd